src/js/views/globals/app-nav/app-nav.scss

```diff
@@ -44,52 +44,117 @@ $menu-hover-color: #375467;
   vertical-align: -0.125em;
 }
 
-.app-nav__title,
-.app-nav__search {
+.app-nav__title {
   font-weight: 700;
   margin-top: 20px;
-  padding: 4px 16px;
+  opacity: 0.5;
+  padding: 4px 16px 8px;
 }
 
 .app-nav__search {
+  cursor: pointer;
+  font-weight: 600;
+  margin-top: 20px;
+  opacity: 0.5;
+  padding: 8px 16px;
+
   .icon {
-    font-size: 12px;
-    margin-right: 8px;
+    font-size: 20px;
+    margin-right: 16px;
     vertical-align: -0.0825em;
+    width: 1em;
   }
 
   &:hover, &.is-active {
     background-color: $menu-hover-color;
     color: $white;
+    opacity: 1;
   }
 }
 
 .app-nav__link {
   cursor: pointer;
-  display: block;
-  padding: 4px 16px;
+  font-weight: 600;
+  opacity: 0.5;
+  padding: 8px 16px;
+  position: relative;
 
   &:hover, &.is-selected {
     background-color: $menu-hover-color;
     color: $white;
+    opacity: 1;
+
+    &::before {
+      background-color: color(light_blue);
+      bottom: 0;
+      content: '';
+      left: 0;
+      position: absolute;
+      top: 0;
+      width: 4px;
+    }
+  }
+
+  &.is-active {
+    background-color: $menu-hover-color;
+    color: $white;
+    opacity: 1;
+  }
+}
+
+.app-nav__link-icons {
+  width: 20px;
+
+  .icon {
+    font-size: 20px;
+    width: 1em;
+  }
+
+  .icon.fa-angle-left {
+    font-size: 16px;
+    width: 8px;
+  }
+
+  .icon.fa-0 {
+    font-size: 16px;
+    width: 10px;
+  }
+
+  .icon.fa-1 {
+    font-size: 16px;
+    width: 14px;
   }
 
-  &.is-selected {
-    font-weight: 700;
+  .icon.fa-3 {
+    font-size: 16px;
+    width: 13px;
```

Comment:
Getting the icons with two combined `<svg>` elements to align correctly was tricky (`<1`, `<3`, `30`).

Each of the numbered icons look slightly different when given differing widths. So each of the `fa-0`, `fa-1`, and `fa-3` icons need different `width` properties for them to align. Or at least that's the only way I could get them to work.